nav/nav_stack/config/global_costmap_params.yaml

```diff
@@ -6,8 +6,41 @@ global_costmap:
   static_map: false     # Not supplying a static map
 
   # Full width and height of IGVC map
-  width: 50
-  height: 50
-  origin_x: -50
-  origin_y: -50
+  width: 200 # Arbitrary large values used for now because I'm not sure if 50 x 50 is actually large enough
+  height: 200
+
+  obstacle_layer:
+    observation_sources: laser_scan_sensor point_cloud_sensor
+    enabled: true
+    track_unknown_space: true
+
+    laser_scan_sensor: {
+      min_obstacle_height: -0.125, # half wheel diameter
+      sensor_frame: base_laser, 
+      data_type: LaserScan, 
+      topic: scan, 
```

Comment:
For topic, I might be remembering wrong but we might have a filtered scan topic, it would be good to update this to use that topic instead

Reply:
I agree, filtered scan topic is better. 

For context, the filtered scan topic does not give the full 360 degree (since behind the lidar is the mount and other stuff) so we only scan like 270 degrees (I think) with filtered scan

Reply:
Scan topic is actually orientated such that the lidar doesn't collide with the robot. Therefore, the filtered scan topic isn't actually used currently.